autoware_launch/launch/logging_simulator.launch.xml

```diff
@@ -108,7 +108,7 @@
 
   <!-- Simulator -->
   <include file="$(find-pkg-share simulator_launch)/launch/simulator.launch.xml">
-    <arg name="scenario_simulation" value="false"/>
+    <arg name="scenario_simulation" value="true"/>
```

Comment:
How about making this an arg? 
https://github.com/tier4/AutowareArchitectureProposal_launcher/blob/2af084219d0fae7effa46ef1277bb8f2082f19c6/autoware_launch/launch/planning_simulator.launch.xml#L69

@KeisukeShima What do you think?

Reply:
If this change is suitable for the use case in CI, I think it is acceptable.

@hayato-m126 What do you think about this suggestion?

Reply:
It's ok, if scenario_simulation defalut value is "true"

```
<arg name="scenario_simulation" default="true" description="use scenario simulation"/>
```

Reply:
I think there is no problem!